src/services/products/billing/billing.ts

```diff
@@ -0,0 +1,48 @@
+import {Logger} from "../../../utils/logger";
+import {BillingAccountResponse, GetPaymentResponse, GetSecretResponse} from "./types/billingResponseTypes";
+import {AxiosInstance, AxiosResponse} from "axios";
+
+export default class BillingSdk {
+  axiosClient: AxiosInstance
+  logger: Logger
+
+  constructor(axiosClient: AxiosInstance, logger: Logger) {
+    this.axiosClient = axiosClient;
+    this.logger = logger;
+  }
+  createBillingAccount(): Promise<BillingAccountResponse> {
+    return this.axiosClient.post<BillingAccountResponse>(
+      `/products/billing/create_billing_account`
+    ).then((res: AxiosResponse<BillingAccountResponse>) => {
+      this.logger.debug(res);
+      return res.data
+    });
+  }
+
+  getSecret(): Promise<GetSecretResponse> {
```

Comment:
This name needs to be improve. Perhaps `getStripeSecret`?

Reply:
yeah that fair